rigex-front-end/src/assets/Tower@1.5x.svg

```diff
@@ -0,0 +1,11 @@
+            <path d="M21.5879638,0.845782857 C21.5879638,0.845782857 23.8261352,2.58365714 20.1134038,6.58603429 C15.4264095,11.668 21.0350038,15.8020343 21.0350038,15.8020343 C28.8817695,7.58662857 21.5879638,0.845782857 21.5879638,0.845782857 Z" id="Path" fill="#ACB9C0" fill-rule="nonzero"></path>
+            <line x1="1.22247048e-12" y1="76.6915657" x2="43.0118561" y2="76.6915657" id="Path-2" stroke="#98A8B1" stroke-width="2.25" stroke-linecap="round"></line>
+        </g>
+    </g>
+</svg>
```

Comment:
please add a newline at the end of this file (not that important, but it removes this little lint warning/error